client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -134,6 +135,10 @@ export class DomainDetailsForm extends PureComponent {
 		return this.state.currentStep !== last( this.state.steps );
 	}
 
+	isInternationalAddress() {
```

Comment:
"International" is a bit ambiguous. I think we're talking about "US" or "nonUS" addresses here - is that right?

Reply:
It's AU/CA/US vs the rest :)

Perhaps something like `isStateRequiredInAddress()` would work since that's the only difference. What do you think?